lib/model/message_list.dart

```diff
@@ -155,6 +155,27 @@ mixin _MessageSequence {
     _processMessage(messages.length - 1);
   }
 
+  /// Given message ids, remove the corresponding messages from the view, and
+  /// return true if at least one message has been removed, or false otherwise.
+  ///
+  /// If none of the message ids provided can be found, this is essentially
+  /// a no-op.
```

Comment:
Here's one idea of how this could be made a little more concise:

```dart
  /// Removes the given messages, if present.
  ///
  /// Returns true if at least one message was present, false otherwise.
  /// If none of [messageIds] are found, this is a no-op.
```

I think all the other relevant facts are implied.